aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -347,7 +362,7 @@ sub is_timeout
     my ($self, $one, $resource, $name) = @_;
     eval {
         local $SIG{ALRM} = sub { die "alarm\n" };
-        alarm TIMEOUT;
+        alarm $TIMEOUT;
```

Comment:
@piojo from http://perldoc.perl.org/functions/alarm.html it seems that you should use `eval/die` to be sure that the timeout will work (but I'm not a perl guru, I'm not sure why..)